src/components/shared/WindowTitle.js

```diff
@@ -85,35 +86,34 @@ function _formatPlatform(meta: ProfileMeta): string {
 }
 
 function _formatDateTime(timestamp: number): string {
-  const timestampDate = new Date(timestamp);
-  let month = timestampDate.getUTCMonth() + 1 + '';
-  let date = timestampDate.getUTCDate() + '';
-  const year = timestampDate.getUTCFullYear();
-
-  if (month.length === 1) month = '0' + month;
-  if (date.length === 1) date = '0' + date;
-
-  const dateLabel = [year, month, date].join('-');
-  const time = _formatTime(timestamp);
-
-  return [dateLabel, time].join(' ');
-}
-
-function _formatTime(timestamp: number): string {
-  const timestampDate = new Date(timestamp);
-  let hours = timestampDate.getUTCHours() + '';
-  let minutes = timestampDate.getUTCMinutes() + '';
-
-  if (hours.length === 1) hours = '0' + hours;
-  if (minutes.length === 1) minutes = '0' + minutes;
-
-  return [hours, minutes].join(':');
+  const dateOptions = {
+    timeZone: 'UTC',
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+  };
+  const timeOptions = {
+    timeZone: 'UTC',
+    hourCycle: 'h24',
+    hour: '2-digit',
+    minute: '2-digit',
+  };
+  const date = new Date(timestamp);
+  const dateLabel = date
+    .toLocaleString('en-GB', dateOptions)
+    .split('/')
+    .reverse()
+    .join('-');
+  const timeLabel = date.toLocaleString('en-GB', timeOptions);
```

Comment:
We really need to use "null" instead of "en-GB" so that it uses the user's locale.